utils.py

```diff
@@ -1940,6 +1955,53 @@ def minimal_realisation(a, b, c):
 
     return Aco, Bco, Cco
 
+def num_denom (A, symbolic_expr = False):
+
+    sym_den = 0
+    sym_num = 0
+    s = sympy.Symbol('s')
+
+    if type(A) == mimotf:
+        denom   = 1
+        num     = 1
+        
+        for j in range(A.matrix.shape[1]):
+            denom = list(numpy.poly1d(denom) * numpy.poly1d(A.matrix[0,j].denominator.coeffs))
+            num   = list(numpy.poly1d(num)   * numpy.poly1d(A.matrix[0,j].numerator.coeffs))
+            if symbolic_expr == True:
+                for n in range(len(denom)):
```

Comment:
This is not very idiomatic python. Have a look at [this video](https://www.youtube.com/watch?v=EnSu9hHGq5o).